amgut/lib/locale_data/british_gut.py

```diff
@@ -89,6 +89,7 @@
 _HANDLERS = {
     'PARTICIPANT_EXISTS': 'Participant %s already exists!',
```

Comment:
Not part of this PR, but I was just noticing that the american and british versions of `_HANDLERS` do not contain exactly the same keys. In particular, the `JUVENILE_CONSENT_EXPECTED` key is missing from this file (not sure about any others). I *would* say that this should lead to bugs when the text locale is assumed to contain exactly the same set of keys, but as far as I can tell the key is never accessed. After verifying that the `JUVENILE_CONSENT_EXPECTED` key is *actually* never accessed, it should probably be removed from the american version. Might also want to verify that, otherwise, the two dicts contain exactly the same keys.